test/unit/test-gesture.js

```diff
@@ -13,6 +15,20 @@ describes.sandboxed('Gestures', {}, (env) => {
     }
   }
 
+  function setPassiveSupported(passiveSupported) {
+    resetPassiveSupportedForTesting();
+    env.sandbox
+      .stub(window, 'addEventListener')
+      .callsFake((name, listener, option) => {
+        passiveSupported ? option.passive : false;
+      });
+    env.sandbox
+      .stub(window, 'removeEventListener')
+      .callsFake((name, listener, option) => {
+        passiveSupported ? option.passive : false;
```

Comment:
Does `removeEventListener` also use the getter trick to test support? I thought that was only done using `addEventListener`?

Reply:
Correct, only 1 of these stubs is needed, so I removed this.